components/atom/tooltip/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@s-ui/react-atom-tooltip",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rm -Rf ./lib && mkdir -p ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@s-ui/component-dependencies": "1",
+    "@researchgate/react-intersection-observer": "0.6.1",
```

Comment:
I think this dependency is not used in the component at all.

Reply:
You're right! Removed